src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/emulated/Cache.java

```diff
@@ -84,7 +85,6 @@
 
     final LongObjectBiConsumer<Runnable> scheduler;
     final Long2ObjectHashMap<Request> correlations;
-    final Supplier<String> etagSupplier;
     final LongSupplier supplyTrace;
     final Int2ObjectHashMap<PendingCacheEntries> uncommittedRequests = new Int2ObjectHashMap<>();
     final Int2ObjectHashMap<PendingInitialRequests> pendingInitialRequestsMap = new Int2ObjectHashMap<>();
```

Comment:
Initialize these `final` fields in the constructor alongside others.
We only typically initialize non-static (per-thread) singletons, such as flyweights, in field definition.